### ISSUE TITLE
Is Julia support available ?

### ISSUE THREAD

**Issue by o314** [joergen7/effi#7]
Hello effi developers ,

I am wondering if it is possible to use effi to push some [julia lang](https://julialang.org) code into an erlang shell ?
*Julia as in the ju of jupyter notebook*

**Comment by joergen7** [comment]
Hey, it is certainly possible. May I ask for the deeper motivation of the request?

**Comment by o314** [comment]
Yes. Julia lacks some kind of a good web framework. I am thinking about something like rack and rake for ruby, 
or mix and plug for elixir. I have started to port some parts of them.
I think erlang, otp has the capacity to be a key stone there (supervisor, etc.).

It would be a great companion to have some ipc in addition to rpc.
So effi could be a could part of this ecosystem. Especially if the cost of this is <= 200 LOC like for python, fs
